src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -124,6 +126,24 @@ public function get(): ?array
             $this->setup();
         }
 
+        $queuedMessageCount = $this->connection->zcount($this->queue, 0, $this->getCurrentTimeInMilliseconds());
```

Comment:
do we need some error handling here? (same for the loop below)

Reply:
Added try catch for it. Thank you for review 👍